pyproject.toml

```
@@ -42,16 +42,17 @@ deprecation = ">=2.1"
 # See https://github.com/readthedocs/readthedocs.org/issues/4912.
 Sphinx = {version=">=5.0", optional=true}
 sphinx_rtd_theme = {version=">=1.0", optional=true}
-grpcio = {version=">=1.49.0,<2.0", optional = true}
-protobuf = { version = "^4.21", optional = true }
+grpcio = {version=">=1.49.0,<2.0", optional=true}
+protobuf = {version="^4.21", optional=true}
 hightime = "^0.2.2"
+toml = {version="^0.10.2", optional=true}
```

Let's use `>=` unless we have evidence that the package uses semver or is otherwise likely to break compatibility.
```suggestion
toml = {version=">=0.10.2", optional=true}
```